cincinnati/src/plugins/internal/edge_add_remove.rs

```diff
@@ -41,6 +41,17 @@ impl PluginSettings for EdgeAddRemovePlugin {
 /// Adds and removes next and previous releases specified by metadata.
 ///
 /// The labels are assumed to have the syntax `<prefix>.(previous|next).(remove|add)=(<Version>,)*<Version>`
+///
+/// # Label processing order
+/// The labels are processed in the following order:
```

Comment:
Maybe `Labels are processed in multiple graph-wide passes, in the following order:`?